sshd/Dockerfile

```diff
@@ -9,6 +9,10 @@ RUN adduser --disabled-password --gecos 'uProxy Giver' --ingroup giver giver
 RUN addgroup getter
 RUN adduser --disabled-password --gecos 'uProxy Getter' --ingroup getter getter
 
+COPY banner /
+RUN chmod 644 /banner
+RUN chown giver: /banner
```

Comment:
The notation is `user:group` but if you omit the group then the user's *primary* group will be used.